.vscode/launch.json

```diff
@@ -1,21 +1,16 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    // https://code.visualstudio.com/docs/editor/debugging#_launch-configurations
```

Comment:
Don't worry about these red highlights in the GitHub UI. GitHub doesn't know that comments are allowed in VS Code JSON config files.